charts/stable/cs2/values.yaml

```diff
@@ -15,7 +15,11 @@ service:
       main:
         protocol: udp
         port: 27015
-
+  rcon:
+    ports:
+      rcon:
+        protocol: tcp
+        port: 27020
```

Comment:
Here its defined as a new service but on questions.yaml is as a new port on the main service.